core/controller/src/main/scala/whisk/core/controller/ApiUtils.scala

```diff
@@ -105,7 +107,7 @@ protected[controller] object FilterEntityList {
  * on an operation and terminate the HTTP request.
  */
 package object PostProcess {
-    type PostProcessEntity[A] = A => RequestContext => Unit
+    type PostProcessEntity[A] = A => RequestContext => Future[RouteResult]
```

Comment:
i see these in several places - can you explain the change for the record? thanks!